pkgs/development/tools/misc/sipp/default.nix

```diff
@@ -1,29 +1,30 @@
-{lib, stdenv, fetchurl, ncurses, libpcap }:
+{ lib, stdenv, fetchurl, ncurses, libpcap, cmake, openssl, git }:
 
 stdenv.mkDerivation rec {
-  version = "3.6.0";
+  version = "3.6.1";
 
   pname = "sipp";
 
   src = fetchurl {
-    url = "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "1fx1iy2n0m2kr91n1ii30frbscq375k3lqihdgvrqxn0zq8pnzp4";
+    url =
+      "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-alYOg6/5gvMx3byt+zvVMMWJbNW3V91utoITPMhg7LE=";
   };
 
-  postPatch = ''
-    sed -i "s@pcap/\(.*\).pcap@$out/share/pcap/\1.pcap@g" src/scenario.cpp
+  configurePhase = ''
+    cmake . -DUSE_GSL=1 -DUSE_PCAP=1 -DUSE_SSL=1 -DUSE_SCTP=0
   '';
 
-  configureFlags = [
-    "--with-pcap"
-  ];
+  buildPhase = ''
+    make -j4
+  '';
 
-  postInstall = ''
-    mkdir -pv $out/share/pcap
-    cp pcap/* $out/share/pcap
+  installPhase = ''
+    mkdir -p $out/bin
+    mv sipp $out/bin
   '';
```

Comment:
I'm having doubts about the install phase being required as well.

Reply:
And I've confirmed it isn't.

Reply:
Hm, not on my side. If I omit it it results in:

```
install flags: SHELL=/nix/store/vssrny41k0mr33jclrrn67rxhbdpwrr0-bash-5.1-p12/bin/bash install
[  0%] Built target version
Consolidate compiler generated dependencies of target sipp
[100%] Built target sipp
Install the project...
-- Install configuration: ""
CMake Error at cmake_install.cmake:41 (file):
  file cannot create directory: /var/empty/local/bin.  Maybe need
  administrative privileges.


make: *** [Makefile:100: install] Error 1
error: builder for '/nix/store/nhfx0678cw62r5sih0giwz2v5k5yk5cr-sipp-3.6.1.drv' failed with exit code 2;
       last 10 log lines:
       > Consolidate compiler generated dependencies of target sipp
       > [100%] Built target sipp
       > Install the project...
       > -- Install configuration: ""
       > CMake Error at cmake_install.cmake:41 (file):
       >   file cannot create directory: /var/empty/local/bin.  Maybe need
       >   administrative privileges.
       >
       >
       > make: *** [Makefile:100: install] Error 1

```

Reply:
Then our cmake defaults likely don't work for your platform?

Reply:
Rechecked and figured I was wrong again. Probably tested too many changes at once.
Works without it on darwin as well.